Makefile

```diff
@@ -8,3 +8,15 @@ DOCKER_RUN=docker run -v $(CURDIR):/mnt otelopscol-build
 .PHONY: package-googet
 package-googet:
 	$(DOCKER_RUN) /bin/bash -c "cd /mnt; make -f .build/Makefile VERSION=$(VERSION) ARCH=$(ARCH) package-googet"
+
+# gcp collector related variables
+IMAGE_NAME=gcpcol_image
+CONTAINER_NAME=gcpcol_container
+
+.PHONY: gcpcol
+gcpcol:
+	docker build -t $(IMAGE_NAME) ./linux
```

Comment:
As per the comment above, while this works, you generally don't want to rebuild your Dockerfile on each build, so would be good to separate this line into a diff make command.

Reply:
I have updated the make file to reuse the build command, and have the gcpcol command just running the image.